libbeat/processors/add_kubernetes_metadata/kubernetes.go

```diff
@@ -31,6 +31,7 @@ func init() {
 	Indexing.AddIndexer(PodUIDIndexerName, NewPodUIDIndexer)
 	Indexing.AddIndexer(ContainerIndexerName, NewContainerIndexer)
 	Indexing.AddIndexer(IPPortIndexerName, NewIPPortIndexer)
+	Indexing.AddIndexer(PodUIDIndexerName, NewPodUIDIndexer)
```

Comment:
Weird, I remember adding it, and it is indeed there on line 31, isn't it? :smiley:

Reply:
uhm, that's correct :man_facepalming: , let me close this, I can confirm it works on master, probably I used a wrong build on my testing